src/ai/backend/manager/api/vfolder.py

```diff
@@ -102,8 +104,20 @@ async def _wrapped(request: web.Request, *args, **kwargs) -> web.Response:
             user_role = request["user"]["role"]
             user_uuid = request["user"]["uuid"]
             allowed_vfolder_types = await root_ctx.shared_config.get_vfolder_types()
-            folder_name = request.match_info["name"]
-            vf_name_conds = vfolders.c.name == folder_name
+            if "id" in request.match_info:
+                folder_id = request.match_info["id"]
+                vf_name_conds = vfolders.c.id == folder_id
+            elif "name" in request.match_info:
+                folder_name = request.match_info["name"]
+                vf_name_conds = vfolders.c.name == folder_name
+            elif isinstance(args[1], dict) and "id" in args[1]:
+                folder_id = args[1]["id"]
+                vf_name_conds = vfolders.c.id == folder_id
+            elif isinstance(args[1], dict) and "name" in args[1]:
+                folder_name = args[1]["name"]
```

Comment:
`args[1]` occurs IndexError when `invitations()` or `list_sent_invitations()` is called
Need to handle the case.

Reply:
FWIW, It seems working properly, but It would be better to categorize the argument received from requests in a structural way. Let's leave it for now and make another issue for that and solve it later.